### ISSUE TITLE
[Bug]: Aborting sync because of net::ERR_HTTP2_PROTOCOL_ERROR

### ISSUE THREAD

**Issue by eazy12** [sboesen/remotely-sync#69]
### What happened?

I cannot perform sync between Linux (Fedora 39 KDE) and Windows 10 using OneDrive. I've made initial sync from Linux today, but I can't manage to sync with Windows or even to sync into completely empty Vault.  The error is net::ERR_HTTP2_PROTOCOL_ERROR and these are my logs:
```
plugin:remotely-secure:180 remotely-secure-1702494364014: abort sync, triggerSource=manual, error while syncing
```
and
```
AggregateError: 
    Error: ML/Probability & Statistics for Engineers & Scientists/standard normal distribution.md: net::ERR_HTTP2_PROTOCOL_ERROR
        at eval (plugin:remotely-secure:112:51897)
    Error: ML/Probability & Statistics for Engineers & Scientists/Hypergeometric Distribution.md: net::ERR_HTTP2_PROTOCOL_ERROR
        at eval (plugin:remotely-secure:112:51897)
    Error: ML/Deep Learning with PyTorch Step-by-Step/Deep Learning with PyTorch Step-by-Step.md: net::ERR_HTTP2_PROTOCOL_ERROR
        at eval (plugin:remotely-secure:112:51897)
    Error: ML/Introduction to Probability, Statistics, and Random Processes/Bernoulli Trial.md: net::ERR_HTTP2_PROTOCOL_ERROR
        at eval (plugin:remotely-secure:112:51897)
    Error: ML/Introduction to Probability, Statistics, and Random Processes/Binomial Formula.md: net::ERR_HTTP2_PROTOCOL_ERROR
        at eval (plugin:remotely-secure:112:51897)
    Error: too many errors, stop the remaining tasks
        at Xre (plugin:remotely-secure:112:52082)
        at async Rx.doActualSync (plugin:remotely-secure:180:4114)
        at async Rx.syncRun (plugin:remotely-secure:180:2769)
    at Xre (plugin:remotely-secure:112:52138)
    at async Rx.doActualSync (plugin:remotely-secure:180:4114)
    at async Rx.syncRun (plugin:remotely-secure:180:2769)
```
I've deleted few similar errors to cencor note names. 

I also had promlems with previous version of add-on (remotely save) but can't name the exact errors. 

### What OS are you using?

Windows, Linux

### What remote cloud services are you using?

OneDrive for personal

### Version of the plugin

0.4.28

### Version of Obsidian

v1.4.16

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by sboesen** [comment]
Odd, I haven't seen this error - looking into our OneDrive sync it looks like we're using fetchUrl from Obsidian, and I don't see an option to override the HTTP protocol.

From googling it [might be related](https://stackoverflow.com/questions/67911570/failed-to-load-resource-neterr-http2-protocol-error-for-react-app-after-upg) to some bug in Windows - there's very little out there for that exact error. 

Is your Windows 10 machine on the latest Windows update?

**Comment by eazy12** [comment]
I've updated my Windows 10 and reloaded and it still downloads the next updates. I've tried to sync to test and it still drops errors. After I've decided to delete folder with PDF files (nearly 1GB of them) from OneDrive and it synced successfully after it. 

I previously had problems with length of PDF files' names (more than 128 characters couldn't be copied into Android) - maybe this is a reason why it was failing. 

The smartphone on Android also finally synced as well.
